src/axes/axis.ts

```diff
@@ -104,6 +104,33 @@ export class Axis<D> extends Component {
     this._scale.offUpdate(this._rescaleCallback);
   }
 
+  /**
+   * Gets the tick label data at a particular point.
+   *
+   * @param {Point} point
+   */
+  public tickLabelAt(point: Point): D {
```

Comment:
this is misnamed - we're not getting the tick label, we're getting the data value corresponding to that tick label